css/main.css

```
@@ -0,0 +1,4 @@
+body {
+  background: #000000;
```

Acho que ficar se colocar somente o zero 3x, tipo assim:

```suggestion
  background: #000;
```